cmd/contour/serve.go

```diff
@@ -418,12 +418,17 @@ func doServe(log logrus.FieldLogger, ctx *serveContext) error {
 				log.WithError(err).Fatal("unable to set up controller manager")
 			}
 
-			// Add the GatetwayAPI Scheme.
+			// Add the Gatetway API Scheme.
```

Comment:
nit: old spelling mistake would be nice to fix

Reply:
I wanted to keep the PR as focused as possible, but I'll go ahead and fix spelling in my upcoming commit.